enterprise/server/execution_search_service/execution_search_service.go

```diff
@@ -35,6 +37,10 @@ type ExecutionSearchService struct {
 	oh  interfaces.OLAPDBHandle
 }
 
+var (
+	enableFetchExecutionRequestMetadata = flag.Bool("enable_fetch_execution_request_metadata", false, "Enable fetching execution request metadata from the OLAP DB")
```

Comment:
we generally prefix these with something so that they can go under a block in yaml (at least `app.`)